recipe/meta.yaml

```diff
@@ -59,3 +71,5 @@ about:
 extra:
   recipe-maintainers:
     - isuruf
+    - tadeu
+    - doxygen
```

Comment:
@doxygen Since you didn't answer, I'm removing your name from the list in order to get this PR ready for merging, but feel free to join at any time :)